qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -167,6 +167,24 @@ def convert_frame_change(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(commands.SetFrequencyInstruction)
+    def convert_set_channel_frequency(self, shift, instruction):
```

Comment:
`channel_` -> `_` here and in the docs `Channel`->``

Reply:
this is later, too, but I'll stop commenting about this

Reply:
I'll do a search for channel. Thanks for pointing this out.

Reply:
Fixed in dfb2b23